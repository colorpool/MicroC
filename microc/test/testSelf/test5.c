void main(int n) {
    match n with
    | 1 -> print 'd' n;
    | 2 -> print 'd' n+1;
    | _ -> print 'd' 0;
}