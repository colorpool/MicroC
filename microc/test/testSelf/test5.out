24 19 1 5 25 13 0 0 1 11 0 1 6 17 25 13 0 0 1 11 22 15 -1 16 35 13 0 0 1 11 0 2 6 17 48 13 0 0 1 11 0 1 1 22 15 -1 16 61 13 0 0 1 11 13 0 0 1 11 6 17 68 0 0 22 15 -1 16 68 15 0 21 0